Failing test: Security Solution Cypress.x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/endpoint_details/insights_ab·cy·ts - Workflow Insights (AB path) should trigger a scan with selected connector should trigger a scan with selected connector

A test failed on a tracked branch

```
AssertionError: Timed out retrying after 60000ms: expected '<button.euiButton.css-1u7ee2e-euiButtonDisplay-s-defaultMinWidth-fill-primary>' to be 'disabled'
    at scanButtonShouldBe (webpack:///./screens/insights.ts:26:49)
    at Context.eval (webpack:///./e2e/endpoint_details/insights_ab.cy.ts:99:6)
```

First failure: [kibana-on-merge - main](https://buildkite.com/elastic/kibana-on-merge/builds/93571#019d7b08-201f-4d06-b0f5-f30cc2859763)

<!-- kibanaCiData = {"failed-test":{"test.class":"Security Solution Cypress.x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/endpoint_details/insights_ab·cy·ts","test.name":"Workflow Insights (AB path) should trigger a scan with selected connector should trigger a scan with selected connector","test.failCount":36}} -->

--- a/x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/endpoint_details/insights_ab.cy.ts
+++ b/x-pack/solutions/security/plugins/security_solution/public/management/cypress/e2e/endpoint_details/insights_ab.cy.ts
@@ -29,8 +29,7 @@ const {
   connectorSelectorExists,
 } = workflowInsightsSelectors;
 
-// Failing: See https://github.com/elastic/kibana/issues/262661
-describe.skip(
+describe(
   'Workflow Insights (AB path)',
   {
     env: {
@@ -94,6 +93,10 @@ describe.skip(
 
       connectorSelectorExists();
       scanButtonShouldBe('enabled');
+
+      // Override pending stub to return running execution BEFORE clicking scan.
+      // Without this, the post-click poll immediately gets [] and completes the scan.
+      fetchRunningWorkflowInsights();
       clickScanButton();
 
       cy.wait('@createWorkflowInsights', { timeout: 30 * 1000 });

